ddtrace/contrib/pyramid/patch.py

```diff
@@ -28,9 +31,13 @@ def traced_init(wrapped, instance, args, kwargs):
     settings = kwargs.pop('settings', {})
     service = os.environ.get('DATADOG_SERVICE_NAME') or 'pyramid'
     distributed_tracing = asbool(get_env('pyramid', 'distributed_tracing', True))
+    analytics = asbool(get_env('pyramid', 'analytics', None))
```

Comment:
Can `asbool()` return `None`?

Reply:
No, I need to handle this with an if-else